### PR TITLE
Add broken tags to FW's that haven't started/stopped correctly in last 3 full runs + script for detecting failing frameworks

### PR DESCRIPTION
**Pull request by ecruz-te**
Python web scraper to detect any frameworks that have failed the past 3 completed runs with the same error

USAGE: `python tfb-fail-detector.py`

### CODE REVIEW

**Review by joanhey — CHANGES_REQUESTED** [review]
Prologue is fixed in #8254

File: frameworks/Nim/prologue/benchmark_config.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
       "database_os": "Linux",
       "display_name": "Prologue",
       "notes": "",
-      "versus": "httpbeast"
+      "versus": "httpbeast",
+      "tags": ["broken"]
```
Comment (joanhey) [inline frameworks/Nim/prologue/benchmark_config.json:22]: Fixed in #8254

**Review by joanhey — CHANGES_REQUESTED** [review]

**Comment by joanhey** [comment]
Please don't close frameworks, without inform their developers.

**Comment by nbrady-techempower** [comment]
> The script is a very good start, like a new useful helper.
> 
> Please add another method to show also the URL to see the error. And perhaps also show the devs, of the fw, in the Techempower repo.

These are some good ideas. Let's iterate on that when we have time.

**Pull request merged**